tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -417,3 +417,15 @@ def test_option_label_from_value_with_placeholder_label(
     actual = placeholder_transform.option_label_from_value(value, answer_id)
 
     assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "trad_as, expected",
+    [
+        ("trad_as", " (trad_as)"),
```

Comment:
I know it sounds silly because it's arbitrary, but maybe make this something other than the variable name in the actual transform to avoid the test passing in case the f-string gets borked and it literally returns `" (trad_as)"`